htdocs/photo.php

```diff
@@ -42,16 +43,25 @@
             error_log("LDAP - Search error $errno  (".ldap_error($ldap).")");
         } else {
             $entry = ldap_get_entries($ldap, $search);
-            if ( !isset($entry[0][strtolower($photo_ldap_attribute)]) ) {
-                if ( $photo_local_ldap_attribute and isset($entry[0][strtolower($photo_local_ldap_attribute)]) ) {
-                    $filephoto = $photo_local_directory . $entry[0][strtolower($photo_local_ldap_attribute)][0] . $photo_local_extension;
-                    if ( file_exists($filephoto) ) {
-                        $photo = imagecreatefromjpeg($filephoto);
+            if ($use_gravatar) {    // If using gravatar
+                $url = "https://www.gravatar.com/avatar/".md5($entry[0]['mail'][0]).".jpg?s=240&d=404";
```

Comment:
You should use $photo_fixed_width and $photo_fixed_height in gravatar URL to define photo size